lib/internal/errors.js

```diff
@@ -1650,6 +1650,9 @@ E('ERR_PERFORMANCE_MEASURE_INVALID_OPTIONS', '%s', TypeError);
 E('ERR_QUIC_CONNECTION_FAILED', 'QUIC connection failed', Error);
 E('ERR_QUIC_ENDPOINT_CLOSED', 'QUIC endpoint closed: %s (%d)', Error);
 E('ERR_QUIC_OPEN_STREAM_FAILED', 'Failed to open QUIC stream', Error);
+E('ERR_REQUIRE_ASYNC_MODULE', 'require() cannot be used on an ESM ' +
+  'graph with top-level await. Use import() instead. To see where the' +
```

Comment:
I think referencing the graph is, whilst completely accurate, too esoteric. I expect many users won't even know what a graph _is_.

Reply:
You mean the word "graph" in general or using "graph" to mean a module and its dependencies? I believe graph is a very common term used across different programming languages, including referring to the concept of "modules and its dependencies". There are also many module graph visualization tools out there, and its used extensively in materials such as [MDN's guide on ESM](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and [V8's user-facing explainer about ESM](https://v8.dev/features/modules). If you Google "javascript module graph" you can find numerous tutorials and blog posts mentioning the concept of module graphs, and I suspect many learned about ESM by reading them so they should already know what this is.